sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -32,7 +30,61 @@ struct RequestWithContinuationToken final
 };
 
 static inline RequestWithContinuationToken BuildRequestFromContinuationToken(
-    Azure::Security::KeyVault::_internal::GetPageResultOptions const& options,
+    Azure::Security::KeyVault::Keys::GetPropertiesOfKeysOptions const& options,
```

Comment:
Oh, the only change here is the rename from `Azure::Security::KeyVault::_internal::GetPageResultOptions` to `Azure::Security::KeyVault::Keys::GetPropertiesOfKeysOptions` and adding three overloads.

nit:
Rather than doing the duplication, maybe have one helper function that accepts `Nullable<NextPageToken>` and `Nullable<MaxPageResults>`, which does all that work?

Reply:
It's implementation detail, I didn't take too much time to think about it. And since we are already duplicating code, it felt fast and easy ahhaha